test/etl_test.py

```diff
@@ -258,14 +257,3 @@ def test_extract_eia923(self, pudl_datastore_fixture):
             raise AssertionError(
                 f"page {page} not found in datastore for {year}"
             )
-
-
-class TestEpaCemsDatastore:
-    """Ensure we can extract csv files from the datastore."""
-
-    def test_get_csv(self, pudl_epacemsdatastore_fixture):
-        """Spot check opening of epacems csv file from datastore."""
-        head = b'"STATE","F'
-
-        csv = pudl_epacemsdatastore_fixture.open_csv("ny", 1999, 6)
-        assert csv.read()[:10] == head
```

Comment:
I suspect this kind of check just ended up somewhere else, but there are a lot of changes and I'm not seeing it immediately. Did it get subsumed in a more general "can you grab a resource from a datastore" test?